docs/page-objects/testing.rst

```diff
@@ -87,10 +91,20 @@ provided by it to generate fixtures in a convenient way.
 .. _scrapy-poet: https://github.com/scrapinghub/scrapy-poet
 .. _Scrapy command: <insert doc link>
 
-
 Running tests
 =============
 
 The provided ``pytest`` plugin is automatically registered when ``web-poet`` is
 installed, and running ``pytest`` in a directory containing fixtures will
 discover them and run tests for them.
+
+.. _web-poet-testing-frozen_time:
+
+Handling time fields
+====================
+
+Page Objects can return the scraping date or even time in the result and it
```

Comment:
There might be other cases when time is important, e.g. page object might use a current timestamp to build some url. What do you think about something along these lines? 

> Sometimes output of a page object might depend on the current time. For example, the item may contain the scraping datetime, or a current timestamp may be used to build some URLs. ...